Analysis/source/SyncTreeProducer.cxx

```diff
@@ -73,14 +75,14 @@ class SyncTreeProducer {
             auto eventInfoPtr = MakeEventInfo(channel, originalTuple.data(), bjet_pair, summaryInfo.get());
             EventInfoBase& event = *eventInfoPtr;
             if(event.GetEnergyScale() != EventEnergyScale::Central) continue;
-            if(args.sample_type() == "data" && !event.GetTriggerResults().AnyAcceptAndMatch(triggerPaths.at(channel)))
-                continue;
+            if(run_period == Period::Run2016 && args.sample_type() == "data" && !event.GetTriggerResults().AnyAcceptAndMatch(triggerPaths.at(channel))) continue;
```

Comment:
no need "data" condition also for 2016: trigger info is present in the last 2016 mc.